source/getting-started/js-primer.md

```diff
@@ -1,14 +1,141 @@
 While the Guides [assume you have a working knowledge of JavaScript](/#toc_assumptions),
 when learning a new framework it can be hard to know what is JavaScript and what is framework,
-especially if you are newer to the ecosystem.
+especially if you are newer to the ecosystem and less familiar with JavaScript.
 
-On top of that, new features were introduced to JavaScript with the release of specification like [EcmaScript 2015](https://developer.mozilla.org/en/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla) (also known as ES6).
-With the current yearly cadence of EcmaScript specifications,
+On top of that, many new features were introduced to JavaScript with the release of newer specifications like ECMAScript 2015 (also known as ECMAScript 6 or ES6, https://developer.mozilla.org/en/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla).
+With the current yearly cadence of ECMAScript specifications,
 developers might have not had time to get familiar with all the new features.
 
 In this guide we will be covering some common JavaScript code patterns that appear in Ember applications,
+and in particular those that explore the new features provided in ES6,
 so you can get a clearer sense of where the language ends and the framework starts.
 
+## `const` and `let` variable declarations
+
+Unlike some programming languages, in JavaScript where your variables are created depends on how you declare them.
```

Comment:
I think "Unlike some programming languages" may be excessive

Reply:
"where your variables are created..."

"In JavaScript, there are a few ways to declare variables that change when and how variables can be used."

Reply:
Thanks @rtablada, all these comments are super helpful. I've updated in a commit just now. Let me know what you think. Flagging for @locks too.